### ISSUE TITLE
Cannot see inserted media (SD)

### ISSUE THREAD

**Issue by webiummedia** [quasar/QuasarRAT#374]
Looks like the file manager does not detect inserted media devices such as phones, SD cards (did not test USB sticks)

**Comment by MaxXor** [comment]
You are right, it's a bit tricky to access the sd-card when you have your phone connected to the PC. USB sticks are working fine.

**Comment by yankejustin** [comment]
@MaxXor Would using WMI allow us to detect network drives, too?

**Comment by MaxXor** [comment]
@yankejustin Network drives are detected already, I have 3.^^

**Comment by yankejustin** [comment]
Oh, good. :)
